R/zzz.R

```diff
@@ -2,5 +2,7 @@ globalVariables(c(
   ".",
   ".data",
   "code",
-  "decode"
+  "decode",
+  "variable",
```

Comment:
I think these are from the select in set_vatiable_label so if you just use select(.data$variable, .data$label) these should go away